app/src/components/main/analytics/events/pluginsPageEvents.js

```diff
@@ -51,18 +51,33 @@ export const getUninstallPluginBtnClickEvent = (pluginName) => ({
 });
 
 const PLUGINS_MODAL = 'Modal plugins';
+const BASIC_PLUGINS_EVENT_PARAMS = getBasicClickEventParameters(PLUGINS_PAGE);
 
 export const PLUGINS_PAGE_EVENTS = {
+  // GA 4
   CLICK_UPLOAD_BTN: {
-    category: PLUGINS_MODAL,
-    action: 'Click on Btn Upload',
-    label: 'Arise Modal Upload Plugin',
-  },
-  OK_BTN_UPLOAD_MODAL: {
-    category: PLUGINS_MODAL,
-    action: 'Click on Btn Ok on Modal "Upload plugins"',
-    label: 'Upload Plugins',
+    ...BASIC_PLUGINS_EVENT_PARAMS,
+    element_name: 'upload',
+    place: 'navigation_panel',
   },
+  UPLOAD_BTN_UPLOAD_MODAL: (type) => ({
+    ...BASIC_PLUGINS_EVENT_PARAMS,
+    element_name: 'upload',
+    modal: 'upload_plugin',
+    type,
+  }),
+  CLICK_CREATE_GLOBAL_INTEGRATION: (type) => ({
+    ...BASIC_PLUGINS_EVENT_PARAMS,
+    element_name: 'button_create',
+    modal: 'create_project_integration',
```

Comment:
It's a little bit confusing that the event name and modal are mismatch each other.
Please recheck with WA. May be its better to name the modal just as `create_integration`.

Reply:
Discussed with WA.
Will be renamed in create_global_integration